### ISSUE TITLE
No logs (in App & syslog server) after upgrading to Homey 5.0 firmware

### ISSUE THREAD

**Issue by pro-sumer** [Dijker/nu.dijker.PaperTrails#43]
**What is your Homey version (Settings - System), your App version (Settings-Apps) ?**

Homey 5.0.0
PaperTrails 0.6.3
Send Log to Developer: No

**What did you try to achieve?**
Log info to syslog server (test flow, with a single action to log a syslog line)

**What was the result?**
* *No* new log line in Log tab of config of App
* *No* new log line on syslog server

**What did you expect as result?**
* New log line in Log tab of config of App
* New log line on syslog server

**Any other remarks**
* This worked fine with PaperTrails 0.6.3 on Homey 4.2.0, until the upgrade.
* Resetting the App did not help (did not try resetting Homey, since I don't know whether that is safe so soon after upgrading)

**Comment by Dijker** [comment]
Hi Rob, here it works since the first rc's and still works in the final.  Even the singel papertrails send to syslog. Can you send a log?

**Comment by pro-sumer** [comment]
No change after a reboot of my Homey.

Log: 48044618-cb79-4eb8-9f82-7503a4a9d6b1

**Comment by Dijker** [comment]
Have you tried to replace the name of the syslog server for the IP address?

**Comment by pro-sumer** [comment]
> Have you tried to replace the name of the syslog server for the IP address?

No, I did not, so I tried now (IP address instead of domain name).

Unfortunately no change:

* no log in App itself
* no log on syslog server

**Comment by pro-sumer** [comment]
* There's no log in the App because that's not in the implementation for "Send syslog msg" (I got confused by that)
* There's no log on the syslog server because Athom changed the reported hostname in the 5.0.0 firmware?

The relevant syslog configuration part on my server:
```
$template HomeyLog, "/var/log/homey.log"
:hostname, isequal, "homey" -?HomeyLog
& stop
```

* This worked with the 4.2.0 firmware which reported `homey` as hostname
* This does not work with the 5.0.0 firmware which reports `homey-<unique ID>` as hostname

So I fixed it by changing my configuration:
```
$template HomeyLog, "/var/log/homey.log"
:hostname, startswith, "homey" -?HomeyLog
& stop
```
